docs/contracts/overview/LSP17ContractExtension.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Extendable Contracts & Extensions Contracts
-sidebar_position: 4
+title: ⚙️ Extendable & Extensions Contracts
```

Comment:
on the sidebar: Extendable & Extension**s** Contracts
on the page title: Extendable & Extension Contracts